lib/index.js

```diff
@@ -102,6 +125,25 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
 
   app.options(accountPath, cors());
 
+  app.get(
+    `${accountPath}/caps`,
```

Comment:
As a general rule we don't shorten APIs -- so this would be `capabilities`. However, we're in the middle of a transition from roles to capabilities and I'd rather not consume the `capabilities` API endpoint with something stale and that uses a bespoke data model and syntax... so let's call this `roles` instead. We can transition to using a `capabilities` endpoint in the future.

In short, please changes this to `/roles`. The call to use `brAccount.getCapabilities` doesn't change.

Reply:
no problem doing it